features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/BusinessInformation/BusinessInformation.jsx

```diff
@@ -53,11 +53,14 @@ const styles = theme => ({
 class BusinessInformation extends React.Component {
     constructor(props) {
         super(props);
+        const {
+            businessOwner, businessOwnerEmail, technicalOwner, technicalOwnerEmail,
+        } = this.props.api.businessInformation;
```

Comment:
i think its better to do 
```
const { api } = this.props
const {businessOwner, businessOwnerEmail, technicalOwner, technicalOwnerEmail, } = api.businessInformation;
```

I think it will throw a eslint error otherwise

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6759/commits/0c218762d38e7db55b9b2b6306a4a1027e6d4cb0